docs/UPGRADE-6.0.md

```diff
@@ -8,7 +8,7 @@ If you find a bug, please open an issue with supporting configuration to reprodu
 - Terraform `v1.5.7` is now minimum supported version
 - AWS provider `v6.0.0` is now minimum supported version
 - The attributes used to construct the container definition(s) have been changed from HCL's norm of `snake_case` to `camelCase` to match the AWS API. There currently isn't a [resource nor data source for the container definition](https://github.com/hashicorp/terraform-provider-aws/issues/17988), so one is constructed entirely from HCL in the `container-definition` sub-module. This definition is then rendered as JSON when presented to the task definition (or task set) APIs. Previously, the variable names used were `snake_case` and then internally converted to `camelCase`. However, this does not allow for [using the `container-definition` sub-module on its own](https://github.com/terraform-aws-modules/terraform-aws-ecs/issues/147) due to the mismatch between casing. Its probably going to trip a few folks up, but hopefully we'll remove this for a data source in the future.
-- `security_group_rules` has been split into `security_group_ingress_rules` and `security_group_egress_rules` to better match the AWS API and allow for more flexibility in defining security group rules.
+- `security_group_rules` has been split into `security_group_ingress_rules` and `security_group_egress_rules` to better match the AWS API and allow for more flexibility in defining security group rules. The `protocol` attribute has been renamed to `ip_protocol`.
```

Comment:
there are more changes than just `protocol` - this is why we have mentioned the previous resource and the new resource. coupled with the new variable attribute definitions, this should be sufficient information that we don't need to re-explain the changes a 3rd time here in this guide